paddle/fluid/framework/paddle2cinn/transform_desc.cc

```diff
@@ -0,0 +1,257 @@
+// Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/framework/paddle2cinn/transform_desc.h"
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "paddle/fluid/framework/paddle2cinn/vartype_utils.h"
+#include "paddle/fluid/platform/enforce.h"
+
+namespace paddle {
+namespace framework {
+namespace paddle2cinn {
+
+using PbVarType = framework::proto::VarType;
+namespace cpp = ::cinn::frontend::paddle::cpp;
+
+void TransformVarDescToCinn(framework::VarDesc *pb_desc,
```

Comment:
Should we make `framework::VarDesc *` as `const framework::VarDesc &` to match the style of `TransformVarDescFromCinn`?

Reply:
Compare to `cpp::Desc`, `framework::Desc` lack of many const API, like `clear()`, `BlockDesc::SetIdx`, etc. On the other hand, the paddle node return `framework::Desc*` even if the node is `const`, so the parameter  in `Transform*DescToCinn` is `framework::VarDesc *` rather than `const framework::VarDesc &`.